books/rtl/rel11/support/harrison.lisp

```diff
@@ -1,12 +1,14 @@
 (in-package "RTL")
 
+(include-book "tools/with-arith5-help" :dir :system)
```

Comment:
No. Certification fails at the end of file processing.
This include only builds table. It doesn't change theory.
Theory is changed later by local events.